store/sheet.go

```diff
@@ -251,16 +251,11 @@ func findSheetList(ctx context.Context, tx *sql.Tx, find *api.SheetFind) (_ []*a
 	}
 
 	// Related fields
-	if v := find.InstanceID; v != nil {
-		where, args = append(where, fmt.Sprintf("instance_id = $%d", len(args)+1)), append(args, *v)
-		if find.InstanceOnly {
-			where = append(where, "database_id is NULL")
-		}
+	if v := find.ProjectID; v != nil {
+		where, args = append(where, fmt.Sprintf("project_id = $%d", len(args)+1)), append(args, *v)
 	}
-	if find.InstanceID == nil || !find.InstanceOnly {
-		if v := find.DatabaseID; v != nil {
-			where, args = append(where, fmt.Sprintf("database_id = $%d", len(args)+1)), append(args, *v)
-		}
+	if v := find.DatabaseID; v != nil {
```

Comment:
Do we have use case of querying over database?

Reply:
Maybe in database transferring, we can use databaseId finding its related sheets to check is it possible to continue transfer in frontend.

Reply:
I see. Let's make a comment for this use case.

Reply:
I add the comment for `SheetFind.DatabaseID` in `./api/sheet.go`.